daemon/execdriver/native/create.go

```diff
@@ -212,6 +212,20 @@ func (d *driver) setupRlimits(container *configs.Config, c *execdriver.Command)
 }
 
 func (d *driver) setupMounts(container *configs.Config, c *execdriver.Command) error {
+	userMounts := make(map[string]bool)
```

Comment:
That's really a detail, but isn't the idiomatic way to implement a `set` in Go to have a `map[string]struct{}` and use `if _, ok := userMounts[m.Destination]; !ok { ... }`?